test/integration/cases_test.go

```diff
@@ -192,6 +192,143 @@ var _ = Describe("Nodepool metrics", func() {
 			)
 		})
 
+		// update node to status ready
+		It("should update first node to ready status", func() {
+			node.Status = corev1.NodeStatus{
+				Conditions: []corev1.NodeCondition{
+					{Type: corev1.NodeReady, Status: corev1.ConditionTrue},
+				},
+			}
+			Expect(k8sClient.Status().Update(ctx, node)).To(Succeed())
+
+			// nodepool_up should be 0; 16x16 topology expects 256
+			By("rechecking the metrics for nodepool_up")
+			nodepool := expectedMetricsForNodePool(np, jsRef.Name, jobRef.Name)
+			assertMetrics(
```

Comment:
If this was down before, it will likely be down now - add a sleep that is longer than the interval

Reply:
done 79ed708